src/examples/Colors/ColorsTemplate.js

```diff
@@ -11,7 +11,9 @@ const ColorsTemplate = () => {
     return <div>
 
         <div className={'p-4 space-y-0.5 '}>
+        
```

Comment:
these changes too shouldn't be part of your PR, you can rebase with master and try to fix this

Reply:
this merge conflict wont allow you to merge, so you'll have to sort this out, make sure this doesn't show up on your PR since it's irrelevant in this PR

Reply:
I have added size prop in the button element and also updated the padding. It is working fine. Getting issue with git branch as you have suggested that with rebase to the main i have tried that way, when i try to make commit in the new branch than the older commits are coming along with the new branch not getting the way how i can isolate that branch from others.

Reply:
you might have branched out from the wrong branch or committed files by mistake, to fix this either

1. You can make a copy of all the file changes you need for this PR, reset your branch to the current master and then copy over the files and commit, you'll be 
2. reset your branch to current main (or probably use a temporary branch for ease), cherry pick the commits one by one til you have all your changes in - this is a bit tricky, but you can give it a try - first way is easier

Reply:
3. You can also check out master, copy the file that has conflict from master, checkout your branch and paste the file in your current branch (probably the easiest fix)